plugins/modules/system/listen_ports_facts.py

```diff
@@ -155,41 +155,59 @@
 from ansible.module_utils.basic import AnsibleModule
 
 
+def split_pid_name(pid_name):
+    try:
+        pid, name = pid_name.split("/")
```

Comment:
How about
```suggestion
        pid, name = pid_name.split("/", 1)
```
so that `1//foo/bar` is split up as `pid == '1'` and `name == '/foo/bar'`?

Reply:
Good idea.  Added